content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -100,6 +101,21 @@ kubectl config unset users.foo                       # delete user foo
 # short alias to set/show context/namespace (only works for bash and bash-compatible shells, current context to be set before using kn to set namespace)
 alias kx='f() { [ "$1" ] && kubectl config use-context $1 || kubectl config current-context ; } ; f'
 alias kn='f() { [ "$1" ] && kubectl config set-context --current --namespace $1 || kubectl config view --minify | grep namespace | cut -d" " -f6 ; } ; f'
+
+# the following aliases can be used to run common commands faster.
+alias kgp='kubectl get pod -owide'
+alias kgs='kubectl get service -owide'
+alias kgd='kubectl get deploy -owide'
+alias kd='kubectl describe'
+alias kg='kubectl get -owide'
+alias ka='kubectl apply -f'
+alias kex='kubectl exec -it'
+alias ke='kubectl edit'
+export dry='--dry-run=client -oyaml'
+export y='-oyaml'
+export w='-owide'
+export j='-ojson'
+export force='--force --grace-period=0'
```

Comment:
These alias are aimed for productivity IIUC.
However, they are also dangerous when users are running Shell scripts.
Having `ka`, `kd`, and even `w`, `y`, `dry` defined is not always a good thing.
I'd suggest we leave these optimizations to users themselves.

Reply:
:+1:

We could _blog_ about these aliases. I don't think they're a good fit for the _cheat sheet_ page.